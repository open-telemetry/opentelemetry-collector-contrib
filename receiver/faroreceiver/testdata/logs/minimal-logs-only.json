{
    "logs": [
        {
            "context": {
                "app": "foo"
            },
            "level": "info",
            "message": "Hello world",
            "timestamp": "2025-03-01T15:42:14Z"
        }
    ],
    "traces": null,
    "meta": {
        "session": {
            "id": "ABCD1234"
        },
        "sdk": {
            "name": "@grafana/faro-core",
            "version": "1.0.0",
            "integrations": []
        },
        "app": {
            "name": "example.com",
            "version": "1.0.0",
            "environment": "prod"
        },
        "view": {
            "name": "default"
        },
        "browser": {},
        "page": {
            "url": "https://example.com/"
        }
    }
  }