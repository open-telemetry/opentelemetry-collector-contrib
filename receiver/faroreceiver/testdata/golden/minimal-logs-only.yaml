resourceLogs:
  - resource:
      attributes:
        - key: service.name
          value:
            stringValue: example.com
        - key: service.version
          value:
            stringValue: "1.0.0"
        - key: deployment.environment
          value:
            stringValue: prod
    scopeLogs:
      - logRecords:
          - body:
              stringValue: "timestamp=2025-03-01T15:42:14Z kind=log message=\"Hello world\" level=info context_app=foo sdk_name=@grafana/faro-core sdk_version=1.0.0 app_name=example.com app_version=1.0.0 app_environment=prod session_id=ABCD1234 page_url=https://example.com/ browser_mobile=false view_name=default"
            attributes:
              - key: kind
                value:
                  stringValue: log